frontend/src/container/BillingContainer/BillingUsageGraph/BillingUsageGraph.tsx

```diff
@@ -71,6 +74,16 @@ export function BillingUsageGraph(props: BillingUsageGraphProps): JSX.Element {
 		data,
 	]);
 
+	const quantityMapArr = useMemo(
+		() => quantityDataArr(graphCompatibleData, chartData[0]),
+		[chartData, graphCompatibleData],
+	);
+
+	useEffect(() => {
+		getCsvData(quantityMapArr);
```

Comment:
why are we setting these on every load? these should be conditional when user clicks on the download csv option